LanguageFeatures/Primary-constructors/static_processing_A03_t02.dart

```diff
@@ -27,22 +27,17 @@ base mixin M4(var int v) {}
 // [analyzer] unspecified
 // [cfe] unspecified
 
-abstract mixin class M5 {
-  this(); // Ok, "trivial" constructor
-}
+abstract mixin class M5(); // Ok, "trivial" constructor
 
-abstract base mixin class M6 {
-  this.someName(var int v);
-//^^^^
+abstract base mixin class M6.someName(final int v) {}
+//                        ^^^^^^^^^^^
 // [analyzer] unspecified
 // [cfe] unspecified
-}
 
-base mixin M7.someName() { // Constructor is "trivial" but it's not a "mixin class"
+base mixin M7.someName(); // Constructor is "trivial" but it's not a "mixin class"
```

Comment:
But we should also have the mixin class with the trivial constructor whose name is two-part, right?

Reply:
Added `M8` with the trivial constructor and in-body part.